docs/docs/guides/creating-your-own-normalizer.md

```diff
@@ -18,16 +18,39 @@ class RoundToWholeNumber(Normalizer, alias="round_numbers"):
         return int(value) if isinstance(value, float) else value
 ```
 
-## Make sure your module is imported
+## Registering your Normalizer
 
-Wherever you have your class defined, nodestream needs to know that its something that should be imported. To do
-so, add your module to the imports section of your `nodestream.yaml` file. For example:
+Normalizers are registered via the [entry_points](https://setuptools.pypa.io/en/latest/userguide/entry_point.html#entry-points-for-plugins) API of a Python Package. Specifically, the `entry_point` named `normalizers` inside of the `nodestream.plugins` group is loaded. Every Value Provider is expected to be a subclass of `nodestream.pipeline.normalizers:Normalizer` as directed above. 
```

Comment:
Is Value Provider overloaded here in these instructions, or am I am not following?  Probably the later...

Reply:
Yeah I copy-pasta-ed here.